src/java/org/apache/cassandra/index/sai/disk/v1/V1OnDiskFormat.java

```diff
@@ -213,6 +211,7 @@ public boolean validatePerColumnIndexComponents(IndexDescriptor indexDescriptor,
                 }
             }
         }
+        long endTime = Clock.Global.currentTimeMillis();
```

Comment:
This seems unused

Reply:
good catch, I forgot to remove it after testing

Reply:
fixed